core/server/worker/src/main/java/alluxio/worker/block/AsyncBlockRemover.java

```diff
@@ -103,9 +106,12 @@ public void run() {
           mBlockWorker.removeBlock(Sessions.MASTER_COMMAND_SESSION_ID, blockToBeRemoved);
           LOG.debug("Block {} is removed in thread {}.", blockToBeRemoved, mThreadName);
         } catch (InterruptedException e) {
-          Thread.currentThread().interrupt();
-          LOG.warn("{} got interrupted while it was cleaning block {}.",
-              mThreadName, blockToBeRemoved);
+          // Only log warning if interrupted not due to a shutdown.
+          if (!mShutdown) {
+            Thread.currentThread().interrupt();
```

Comment:
is this supposed to be in the if statement too?

Reply:
Right. This was incorrect.

Reply:
https://github.com/Alluxio/alluxio/pull/10777/commits/275c3d201fb0820e3222cc7e6349f703224b5146